.github/workflows/conformance-eks-v1.11.yaml

```diff
@@ -222,6 +237,11 @@ jobs:
             until docker manifest inspect quay.io/${{ github.repository_owner }}/$image:${{ steps.vars.outputs.sha }} &> /dev/null; do sleep 45s; done
           done
 
+      - name: Checkout code
```

Comment:
Why second checkout?

Reply:
Same for other workflows. Might be worth adding a comment why it's needed?

Reply:
@tklauser I think this is the only workflow in which there were two checkouts in same job? AFAIU all added checkouts are needed so that we have available helm charts that are stored in `install/kubernetes` dir

Reply:
Yeah, not all other workflows. But I think the `.github/workflows/conformance-externalworkloads*.yml` jobs also have a second checkout in the same job.

Reply:
Nevermind, these are changed to `ref: <branch>`. So all good apart from the one @nebril pointed out.

Reply:
Ups my bad, I'll remove it. It's odd, even for the other jobs there shouldn't be a duplicate "check out"